README.md

```diff
@@ -83,6 +82,10 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v1
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0.3
```

Comment:
It's kind of a breaking change since this change compels users to define this section.